### PR TITLE
Add filter fulfillment order line items example code

### PR DESCRIPTION
**Pull request by JacobyJoukema**
## Overview

Resolves https://github.com/Shopify/core-issues/issues/66261

## Discussion

The following code sample is built to demonstrate filtering and splitting fulfillment orders. It can be used in a workflow such as the one shown below:

image

## Testing

Tophatted in production using the workflow shown above

All tests pass
image

### CODE REVIEW

**Review by plnii — CHANGES_REQUESTED** [review]
Definitely change folder name. Open to how you approach testing.

File: run-code-examples/filter-fulfillment-order-line-items/index.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// Filter line items based on merchants desired parameter
+function shouldSplitLineItem(orderLineItems, fulfillmentOrderLineItem) {
+   return orderLineItems.some(lineItem => {
+    return lineItem.variant.product.title === fulfillmentOrderLineItem.productTitle && lineItem.variant.product.tags.includes("Refrigerated");
```
[the remark this answers is not in the document]
Comment (JacobyJoukema) [reply run-code-examples/filter-fulfillment-order-line-items/index.js]: Updated to the following:
```
// This function filters fulfillmentOrderLine items if the product has a specific tag
// This function can be updated to filter based on a different condition or parameter
```

File: run-code-examples/filter-fulfillment-order-line-items/index.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+  if (fulfillmentOrderSplitLineItems.length >= 0) {
+    fulfillmentOrderSplits.push({
+      "fulfillmentOrderId": fulfillmentOrder.id,
+      "fulfillmentOrderLineItems": fulfillmentOrderSplitLineItems,
+    })
+  }
+
+  return {
+    fulfillmentOrderSplits: JSON.stringify(fulfillmentOrderSplits, null),
```
[the remark this answers is not in the document]
Comment (JacobyJoukema) [reply run-code-examples/filter-fulfillment-order-line-items/index.js]: Good catch! Thanks Steve

**Review by plnii — APPROVED** [review]
LGTM

**Pull request merged**